plugin_tests/pycoverage.cfg

```diff
@@ -0,0 +1,10 @@
+[run]
+branch = False
+
+[report]
+# omit = girder/!(plugins)/*,clients/python/*,girder/plugins/!(osumo)/*
+# omit = girder/*,clients/python/*,girder/plugins/!(osumo)/*
+omit = girder/*,clients/python/*
+# include = girder/plugins/osumo/*
+# include = server/*
+
```

Comment:
No -- I'll remove them.  But I'm happy to have them in git history in case I want to see what I tried.